nodemon.json

```diff
@@ -0,0 +1,11 @@
+{
+  "watch": [
+    "bin/",
+    "config/",
+    "src/server/",
+    "src/client/core",
+    "src/client/middlewares",
+    "src/client/",
```

Comment:
Do we want to watch client? Wouldn't that restart the express server for client js changes? Webpack hot-reload should take care of that